latch/resources/tasks.py

```diff
@@ -87,7 +87,9 @@ def _get_large_pod() -> Pod:
     primary_container.resources = resources
 
     return Pod(
+        annotations={"io.kubernetes.cri-o.userns-mode": "auto:size=65536"},
```

Comment:
what does the `auto:size=65536` do? assuming it specifies memory, will it supercede the memory constraints specified by the task type?

Reply:
https://github.com/cri-o/cri-o/blob/main/tutorials/userns.md

It has nothing to do with memory.

"io.kubernetes.cri-o.userns-mode": Tells CRI-O to launch this as a rootless pod (i.e., root user in the pod maps to an unprivileged user on the host) and to allocate a range of 65536 Linux user-namespace user and group IDs. This is required for Sysbox pods.